### PR TITLE
Cd sprint 1 functionality

### PR DESCRIPTION
**Pull request by chadrick-d-dev**
## Description
- Updated Readme
- Commented out the data imports in Recipe/User-test
- Refactored User-test, passing all tests on user.js

## Motivation and Context
- Initially we had errors warning us that webpack could not read our test files when we ran ```npm test```
  - We remedied this by creating a sample data set within our test file.

## How Has This Been Tested?
- We used a mocha framework and chai assertion library to test our files
- This affects our user class and checks for its functionality.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All existing tests passed.

### CODE REVIEW

**Comment by ericberglund117** [comment]
I like the concise and specific atomic commits. I can really tell what you did to the user-test.js based on the commit messages

File: test/user-test.js [hunk 1]
```diff
@@ -1,16 +1,73 @@
...
-  let user;
+  let user1;
+  let user2;
   let userInfo;
   let recipe;
 
   beforeEach(function() {
-    userInfo = data.users[0];
-    user = new User(userInfo)
+    userInfo = [
```
Comment (Kristiannmiller) [inline test/user-test.js:13]: Let's revisit the amount of information we have in our sample dataset once we've gone through the tests. We might be able to narrow it down a bit later.

File: test/user-test.js [hunk 2]
```diff
@@ -20,42 +77,42 @@ describe('User', function() {
...
-    user.saveRecipe(recipe);
-    expect(user.favoriteRecipes[0].name).to.equal('Chicken Parm');
+    user1.saveRecipe(recipe);
+    expect(user1.favoriteRecipes[0].name).to.equal('Chicken Parm');
   });
 
   it('should be able to decide to cook a recipe', function() {
-    user.decideToCook(recipe);
-    expect(user.recipesToCook[0].name).to.equal('Chicken Parm');
+    user1.decideToCook(recipe);
```
Comment (Kristiannmiller) [inline test/user-test.js:105]: Good catch on this rename!

**Review by ericberglund117 — APPROVED** [review]

**Comment by Kristiannmiller** [comment]
Looks great! Let's keep going with this functionality and maybe do some sad path testing for this class on our next branch

**Pull request merged**